src/components/Metadata.vue

```diff
@@ -1,22 +1,53 @@
 <template>
-  <aside class="metadata-container u-flex">
-    <h3 class="work-title">{{ metadata.workTitle }}</h3>
-    <tt class="work-urn">{{ metadata.workUrn }}</tt>
-  </aside>
+<div class="metadata-container u-flex">
```

Comment:
![image](https://user-images.githubusercontent.com/629062/85802118-e010af80-b709-11ea-9e17-55c46d754269.png)

Reply:
- [ ] Revisit https://github.com/scaife-viewer/explorehomer/pull/38#issuecomment-649763924

Reply:
Going to punt on the wrapping for now